src/components/Hero/Hero.mdx

```diff
@@ -198,6 +201,55 @@ subHeaderText={otherSubHeaderText}
 
 <Canvas of={HeroStories.CampaignBackgroundColors} />
 
+## Fallback Campaign Image
```

Comment:
Can this same approach be used for the `"primary"` variant?  What about the `FeaturedContent` component?

Reply:
Yes, I was focusing on the needed variant first before doing the rest but I can work on updating the "primary" variant and have an example. How about `FeaturedContent` as it's own example? Right now the raised issue is with the `Hero` since that uses dynamic images but `FeaturedContent` does not (in DC Facelift).

Reply:
Actually, I can't! No image is loaded for the `primary` variant so it's not possible to check for an image loading error. "primary" only uses the background-image CSS property. We'll need a different solution for this...

I'd say not for this PR.

Reply:
The solution that is suggested here for DC facelift is only possible because they use the same image in the foreground and background.  Technically, the solution that is eventually used for `primary` should also be used here.  But yes, I agree, that should be handled in a separate ticket.